project-base/app/tests/FrontendApiBundle/Functional/Complaint/CreateComplaintTest.php

```diff
@@ -65,18 +73,42 @@ public function testCreateComplaint(): void
         $this->assertArrayHasKey('items', $responseData);
         $this->assertCount(2, $responseData['items']);
 
-        $this->assertArrayHasKey('quantity', $responseData['items'][0]);
-        $this->assertSame(1, $responseData['items'][0]['quantity']);
-        $this->assertArrayHasKey('files', $responseData['items'][0]);
-        $this->assertCount(2, $responseData['items'][0]['files']);
-        $this->assertArrayHasKey('orderItem', $responseData['items'][0]);
-        $this->assertArrayHasKey('name', $responseData['items'][0]['orderItem']);
+        $this->assertComplaintItem(
+            $responseData['items'][0],
+            $complaintItemQuantity1,
+            $complaintItemFilesCount1,
+            $product1,
+        );
+
+        $this->assertComplaintItem(
+            $responseData['items'][1],
+            $complaintItemQuantity2,
+            $complaintItemFilesCount2,
+            $product2,
+        );
+    }
 
-        $this->assertArrayHasKey('quantity', $responseData['items'][1]);
-        $this->assertSame(2, $responseData['items'][1]['quantity']);
-        $this->assertArrayHasKey('files', $responseData['items'][1]);
-        $this->assertCount(1, $responseData['items'][1]['files']);
-        $this->assertArrayHasKey('orderItem', $responseData['items'][1]);
-        $this->assertArrayHasKey('name', $responseData['items'][1]['orderItem']);
+    /**
+     * @param array $expectedComplaintItem
+     * @param int $quantity
+     * @param int $filesCount
+     * @param \App\Model\Product\Product|null $product
+     */
+    private function assertComplaintItem(
+        array $expectedComplaintItem,
+        int $quantity,
+        int $filesCount,
+        ?Product $product = null,
+    ): void {
+        $this->assertArrayHasKey('quantity', $expectedComplaintItem);
+        $this->assertSame($quantity, $expectedComplaintItem['quantity']);
+        $this->assertArrayHasKey('files', $expectedComplaintItem);
+        $this->assertCount($filesCount, $expectedComplaintItem['files']);
+        $this->assertArrayHasKey('orderItem', $expectedComplaintItem);
+        $this->assertArrayHasKey('name', $expectedComplaintItem['orderItem']);
+        $this->assertArrayHasKey('productName', $expectedComplaintItem);
+        $this->assertSame($product?->getName($this->domain->getLocale()), $expectedComplaintItem['productName']);
```

Comment:
`$this->domain->getLocale()` is unnecessary if I am not mistaken

Reply:
Fixed @KennyDaren
[fixup! complaints b2b customers support](https://github.com/shopsys/shopsys/pull/3295/commits/585c5b09e615c10c4351fc201d7abf686fa2e022)